monkey/infection_monkey/system_info/mimikatz_collector.py

```diff
@@ -45,12 +47,16 @@ def init_mimikatz(self):
             collect_proto = ctypes.WINFUNCTYPE(ctypes.c_int)
             get_proto = ctypes.WINFUNCTYPE(MimikatzCollector.LogonData)
             get_text_output_proto = ctypes.WINFUNCTYPE(ctypes.c_wchar_p)
+            T1106Telem(ScanStatus.USED, "WinAPI was called to load mimikatz.").send()
```

Comment:
I think this should be sent as USED iff T1129Telem is sent as USED.
You can just check `self._isInit` after the try-except clause, and send appropriate telems for both techniques